include/zephyr/bluetooth/services/ans.h

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2025 Sean Kyer
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_SERVICES_ANS_H_
+#define ZEPHYR_INCLUDE_BLUETOOTH_SERVICES_ANS_H_
+
+/**
+ * @brief Alert Notification Service (ANS)
+ * @defgroup bt_ans Alert Notification Service (ANS)
+ * @ingroup bluetooth
+ * @{
+ */
+
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief ANS max text string size
+ *
+ * This is the max string length to be saved in a New Alert. Text longer than the max is truncated.
+ */
+#define BT_ANS_MAX_TEXT_STR_SIZE 18
+
+/**
+ * @brief ANS Category ID Type
+ */
+typedef uint8_t bt_ans_cat_t;
+
+/**
+ * @brief ANS Category ID Enum
+ *
+ * Enumeration for whether the category is supported.
+ */
+enum {
+	bt_ans_cat_simple_alert,
+	bt_ans_cat_email,
+	bt_ans_cat_news,
+	bt_ans_cat_call,
+	bt_ans_cat_missed_call,
+	bt_ans_cat_sms_mms,
+	bt_ans_cat_voice_mail,
+	bt_ans_cat_schedule,
+	bt_ans_cat_high_pri_alert,
+	bt_ans_cat_instant_message,
+	/* Marker for number of categories */
+	bt_ans_cat_num,
+	/* 10–15 reserved */
+};
+
+/**
+ * @brief Send a new alert to central
```

Comment:
Suggest to change `central` to `remote devices` or something similar. 

The reason for this is that
1) The service may exist on a central device, and the peripheral may be the one who subscribed to the alerts
2) The service probably need to notify all connected clients